WordPress/gutenberg
# Fixtures: Regenerate Latest Posts server fixtures

Related: #3745 (60c7158a729bba332ed5361c1d676a8de690c997)

This pull request seeks to regenerate fixtures, which had fallen out of date with changes introduced in #3745, but apparently in such a way that did not trigger build failure. This avoids unintended changes which can occur when running `npm run fixtures:regenerate` on the master branch. The changes introduced by 60c7158a729bba332ed5361c1d676a8de690c997 rename the Latest Posts block's `layout` attribute to `postLayout`, which can be seen reflected here.

__Testing instructions:__

Verify there are no test failures:

```
npm test
```

Verify there are no changes introduced when regenerating fixtures:

```
npm run fixtures:regenerate
```
